### DISCUSSION TITLE
How should I use it

### DISCUSSION THREAD

**Discussion by xaoyaoo** [rotki/pysqlcipher3#8] [Q&A]

**Comment by kelsos** [comment]
Hey @xaoyaoo, I am not sure what do you mean? 

This package's purpose is to each the developer experience for rotki, and also speedup building and packaging.

It only builds and publishes wheels for the version of python and OSes that rotki uses/supports.

What are you looking for exactly?

**Comment by xaoyaoo** [comment]
I am looking for a compilation and installation method for pysqlcipher3 because I always encounter various errors

**Reply by kelsos** [reply]
So If you are using python 3.10 and you need to support the same OSes as rotki 

You need just need to add it to requirements:
https://github.com/rotki/rotki/blob/972506a269fd0c1937194ab5de96daabda870f53/requirements.txt#L7

And import as normal `pysqlcipher3`.

There is a previous version that supports Python 3.9 but has an issue where OpenSSL is not statically linked for macOS.

If you need a newer version of Python, what you could do is fork this repo, adjust it for building wheels for this version and publish on PyPI. 

We don't have any direct plans of upgrading Python, although we will probably do at some point in the future.
